kafka-connector/src/main/java/com/google/pubsub/kafka/sink/CloudPubSubSinkTask.java

```diff
@@ -405,6 +408,7 @@ private void createPublisher() {
                     .setInitialRpcTimeout(Duration.ofSeconds(10))
                     .setRpcTimeoutMultiplier(2)
                     .build())
+            .setExecutorProvider(FixedExecutorProvider.create(getSystemExecutor()))
```

Comment:
This changes the current executor which is num_cpus*5 https://github.com/googleapis/java-pubsub/blob/0bba01661c01ca540625f63a0d7e89d43e2dab34/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java#L683-L687. 

This might reduce throughput for high-volume sinks.